### ISSUE TITLE
[New feature]

### ISSUE THREAD

**Issue by sdrogers** [mwang87/MetabolomicsSpectrumResolver#80]
Request from Madeleine: can we show ppm errors for matched peaks in mirror plots?

**Comment by bittremieux** [comment]
I'd have to think a bit about how to report that. I'm not sure how to do that nicely in the plot, so maybe a table with matched peak m/z's and ppm errors instead?

**Comment by sdrogers** [comment]
Yes, I think that would probably be neater....nice idea. And easier :-)
